public/sass/pages/_dashboard.scss

```diff
@@ -102,22 +102,22 @@ div.flot-text {
 
   .fa {
     position: relative;
-    top: -2px;
+    top: -4px;
     left: -5px;
-    font-size: 90%;
+    font-size: 75%;
   }
 
   &--info {
     display: block;
-    background: $blue-dark;
+    background: mix($panel-bg, $body-bg, 35%);
```

Comment:
not sure if it's a good idea mixing colors, basically sneaking in a another color, is there no color we can use from the palette?